Assets/DeathMario.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public sealed class DeathMario : DeathBase
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+
+    }
+
+    // Update is called once per frame
+    protected new void Update()
+    {
+        base.Update();
+    }
+
+
+    public override void Die()
+    {
+        if (_isDead == false)
+        {
+            // 演出
+            var rb = this.GetComponent<Rigidbody2D>();
```

Comment:
[SerializeField] をつけて、あらかじめアタッチできないか、検討をお願いします。
```suggestion

[SerializeField] 
    private RigidBody2D _rb = null;
、、、
            _rb = this.GetComponent<Rigidbody2D>();
```